cli/integration/suite_test.go

```diff
@@ -0,0 +1,177 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/log"
+	"github.com/testcontainers/testcontainers-go/modules/registry"
+
+	"ocm.software/open-component-model/bindings/go/oci"
+	urlresolver "ocm.software/open-component-model/bindings/go/oci/resolver/url"
+	"ocm.software/open-component-model/cli/integration/internal"
+)
+
+// TestSuite holds shared test infrastructure
+type TestSuite struct {
```

Comment:
This needs different names and a different setup because it works only for OCI registries